modules/tensor_mechanics/src/materials/ComputeLinearElasticStress.C

```diff
@@ -16,6 +16,7 @@ InputParameters validParams<ComputeLinearElasticStress>()
 
 ComputeLinearElasticStress::ComputeLinearElasticStress(const InputParameters & parameters) :
     ComputeStressBase(parameters),
+    _stress_old(declarePropertyOld<RankTwoTensor>(_base_name + "stress")),
```

Comment:
Same thing here. Do this in a class that is derived from ComputeLinearElasticStress. Adding stateful materials adds a **substantial** memory overhead to a material! And we do not need them for many applications.